public/index.html

```diff
@@ -1,23 +1,17 @@
-<html lang="pl">
-    <head>
-        <meta charset="utf-8">
-        <title>Praktyki</title>
-        <link href="https://unpkg.com/tailwindcss@^2/dist/tailwind.min.css" rel="stylesheet">
-    </head>
-    <body class="min-h-screen flex flex-col">
-        <div class="flex flex-1 p-16">
-            <div class="container mx-auto">
-                <div class="w-full md:w-1/2 p-16 text-white text-center bg-gradient-to-r from-purple-500 to-purple-600 rounded-xl shadow-xl text-2xl">
-                    {{ <strong>tu znajdzie się zbudowana strona</strong> }}
-                </div>
-            </div>
-        </div>
-        <footer class="bg-white">
-            <div class="mx-auto py-12 px-4 overflow-hidden">
-                <p class="text-center text-base text-gray-400">
-                    &copy; 2021 Blumilk
-                </p>
-            </div>
-        </footer>
-    </body>
+<!DOCTYPE html>
+<html lang="">
```

Comment:
Please add the language code here:
```suggestion
<html lang="pl">
```